yagpdb-cc-website/docs/snippets/parseflags.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 11
+title: parseArgs
+---
+
+This snippet provides a reusable template which you can add to your custom commands.  
+It separates predefined flags from positional arguments within input; for example:
+- `-command -m 123 positional arg -f`
+    - might be parsed into:
+    
+    positional: ["positional" "arg"]  
+	  flags: map {  
+	    "m" => "123"  
+	    "f" => true  
+	  }
```

Comment:
```suggestion
    ```
    positional: ["positional" "arg"]  
	  flags: map {  
	    "m" => "123"  
	    "f" => true  
	  }
	  ```
```
could we get a codeblock here?